src/main/java/com/gojek/beast/converter/fields/DateField.java

```diff
@@ -0,0 +1,33 @@
+package com.gojek.beast.converter.fields;
+
+import com.google.cloud.Date;
+import com.google.protobuf.Descriptors;
+import com.google.protobuf.DynamicMessage;
+import lombok.AllArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+public class DateField implements ProtoField {
+    private final Descriptors.FieldDescriptor descriptor;
+    private final Object fieldValue;
+
+    @Override
+    public Object getValue() {
```

Comment:
@anukin Could we add test for this getValue

Reply:
This is done